### PR TITLE
OLCNE-5263: multus module

### PR DESCRIPTION
**Pull request by guoyongzhang**
https://jira.oci.oraclecorp.com/browse/OLCNE-5263

### CODE REVIEW

File: module-operator/controllers/module/chartutil.go [hunk 1]
```diff
@@ -55,14 +55,19 @@ func lookupChartLeafDirName(mod *moduleapi.Module) string {
 	switch mod.Spec.ModuleName {
 	case string(moduleapi.CalicoModuleClass):
 		if version == "" {
-			version = "3.25.0"
+			version = "3.25.1"
```
Comment (pfmackin) [inline module-operator/controllers/module/chartutil.go:58]: why did you change the Calico version?

Comment (guoyongzhang) [reply module-operator/controllers/module/chartutil.go:58]: So 3.25.1 is the default

**Review by pfmackin — APPROVED** [review]
